Misc/NEWS.d/next/Library/2017-12-22-16-05-01.bpo-32410.8JzhvH.rst

```diff
@@ -0,0 +1 @@
+Implement ``loop.sock_sendfile`` for asyncio UNIX event loop.
```

Comment:
Given that we implement a fallback, it's not just for Unix anymore.  Change to "Implement loop.sock_sendfile".